docs/how-to/manage_service_accounts.md

```diff
@@ -1,8 +1,8 @@
-### How-To Guide
+## Manage Spark Client Accounts
 
 This section provides general commands on how to use the CLI interface for creating, managing and configuring spark applications.
 
-#### Create Service Account
+### Create Service Account
 
```

Comment:
provide an explainer of what the subsequent command is going to do

Reply:
Isn't this obvious from the title of the section? A how-to can also be as dry as possible just for the sake of being direct to the point